### PR TITLE
feat: enable rouge highlighter

### PR DESCRIPTION
**Pull request by willoucom**
After discussion in PR #33  I decided to do some modifications based on my workaround.

The modifications on the first blog post is only for show and can be (and will be) removed upon request.

The strip_newlines filter on `page.content` prevents the bloc-quotes and highlighter to work properly, I don't know why it was added (obviously for a good reason), so I removed it.

Thanks a lot for your review.

### CODE REVIEW

File: _posts/2019-01-29-hello-world.md [hunk 1]
```diff
@@ -1,8 +1,75 @@
 ---
 title: "Welcome to Jekyll!"
-published: false
+published: true
```
Comment (drewbrew) [inline _posts/2019-01-29-hello-world.md]: Did you mean to change published to true here?

Comment (willoucom) [reply _posts/2019-01-29-hello-world.md]: In fact, I am not very confident with this change. 
At first i thought this was a good idea to publish a first post with examples of highlighter. Besides this modification can enable an easier review of this PR. But some users may not want a first post, i don't know for the moment.
If you suggest to revert the published state to "false", i wouldn't mind doing it.

**Comment by brandonrosage** [comment]
I've added a commit that improves the default styles for the body content in the blog post. 👍  And I've restored the post's `published` status to `false`.

At least at this early stage, the blogging use case is very secondary. I'm not sure yet how prominently that'll get used, and I'd prefer not to force folks to turn things off. I think secondary features are better suited for a "turn it on" experience.

What do you think?

**Comment by willoucom** [comment]
About the examples, many themes use a "demo" page to display (for example: https://github.com/pages-themes/midnight/blob/master/index.md)
Maybe, in another PR, we can use the "_draft" feature to expose a demo page ? (source: https://jekyllrb.com/docs/posts/#drafts)

**Comment by brandonrosage** [comment]
Yeah, I'd just like to be a little more thoughtful about the default. One of the design principles for the repo was that you could fork it and, without changing anything, have a functioning, ready-to-go website. So I wanted to hide as much "placeholder" content from that public build.

Make sense?

**Pull request merged**